manager/src/main/resources/db/migration/0.1/V0.1.3__create_filter.sql

```diff
@@ -0,0 +1,10 @@
+create table FILTER
+(
+    id                 varchar(255) NOT NULL PRIMARY KEY,
+    processor_id       varchar(255) NOT NULL,
+    key                varchar(255) NOT NULL,
+    type               varchar(255) NOT NULL,
+    value              TEXT NOT NULL,
+    version            integer NOT NULL default 0,
```

Comment:
I don't see the version column mapped in the `Filter` entity. Not 100% sure we need it on Filters. It's there to support Optimistic locking in JPA (see https://vladmihalcea.com/jpa-entity-version-property-hibernate/), so it's gonna depend a little bit on our strategy of updating Processors in the future, but I'd expect the `version` on the Processor to be enough.

Reply:
Removed